icons/svg/file_type_jsconfig.svg

```diff
@@ -0,0 +1,13 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
This is just another icon for ordinary JavaScript files for which icons exist already. I don't see benefit of adding alternate icons for jsconfig files, which don't indicate being a config file. While the icon itself looks great, it somehow feels off along with the style used by all the other more minimalistic ones.

Same applies to file_type_tsconfig.